scipy/fft/_pocketfft/pypocketfft.cxx

```diff
@@ -158,7 +158,7 @@ template<typename T> py::array c2c_sym_internal(const py::array &in,
     iter.advance();
     }
   }
-  return move(res);
+  return std::move(res);
```

Comment:
LGTM at least, though I'm not a C++ expert. It seems this has been available since C++11, which looks reasonably safe based on our C++ toolchain section: https://scipy.github.io/devdocs/dev/toolchain.html#c-language-standards